src/transformers/models/auto/configuration_auto.py

```diff
@@ -105,6 +106,7 @@
         ("mpnet", "MPNetConfig"),
         ("mt5", "MT5Config"),
         ("mvp", "MvpConfig"),
+        ("nat", "NATConfig"),
```

Comment:
Can you also add the config archive maps in the constant below?

Reply:
Done.